source/features/easy-toggle-commit-messages.tsx

```diff
@@ -18,13 +18,13 @@ function toggleCommitMessage(event: DelegateEvent<MouseEvent>): void {
 		return;
 	}
 
-	$('.ellipsis-expander', event.delegateTarget)?.dispatchEvent(
+	$('[data-testid="commit-row-show-description-button"]', event.delegateTarget)?.dispatchEvent(
 		new MouseEvent('click', {bubbles: true, altKey: event.altKey}),
 	);
 }
 
 const commitMessagesSelector = [
-	'.js-commits-list-item',
+	'[data-testid="commit-row-item"]',
 	':is(.file-navigation, .js-permalink-shortcut) ~ .Box .Box-header', // Commit message in file tree header
```

Comment:
Does the second selector still work? We should probably also include the other test URLs below. See https://github.com/refined-github/refined-github/pull/5526 for context

Reply:
New URLs:

- Repo root: https://github.com/jpmens/jo/tree/eefaf71b4106f791b74bd21eaea2d27f9cd03ce1
- File/folder: https://github.com/jpmens/jo/tree/eefaf71b4106f791b74bd21eaea2d27f9cd03ce1/tests

File and folder have the same format, no need to duplicate the URL